src/components/network/NetworkItem.vue

```diff
@@ -14,6 +14,11 @@
         {{ networkitem.chain_id }}
       </p>
     </div>
+    <PoweredBy
+      :network="networkitem"
```

Comment:
this is not currently working in production.

Reply:
yes, also not working here https://develop--lunieio.netlify.com/networks

I am looking into it... Why in the netlified branch it works but not in netlify develop?

Reply:
Fixed 🎉  ! #3653 3653